### PR TITLE
replace how we get image version for artist

### PR DESCRIPTION
**Pull request by oxaudo**
Putting this in it's own super tiny pr. Actually not 100% sure if it would be good to still keep cropped here? The artist does not seem to have 140px square version and the square is 200px. 

@mzikherman - what is the proper image here would you happen to know? It's for the artist.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 874b164b7e293fe138811a11759ebdefab311981

https://deploy-preview-1368--artsy-reaction.netlify.com

**Comment by oxaudo** [comment]
Yea, I guess my real question was if there is a difference in performance between on the fly resizing and using pre-resized versions. We seem to use both in code pretty equally, so not sure where to lean.

**Comment by mzikherman** [comment]
Perf wise on the backend there is no difference, we should feel free to use whatever.

On the frontend/mobile devices with poor internet, I think you basically want something thats resized for the exact size being initially rendered, ideally. However, the main bug is just to avoid sending out giant originals/unmodified ones, so these little square ones are probably fine.

Some components (esp in Emission), like the grid, actually use the following strategy: https://github.com/artsy/reaction/blob/b28f00565c7c9480e43a33601c2d7f60b461bdff/src/Components/Artwork/GridItem.tsx#L73 .

Basically, a specific version is requested (large), and then you use the actual breakpoints to set the URL to a resized URL (so rather than request a resized value from MP, the component decides how to resize).

That approach makes more sense for Emission and image heavy components like the grid, I think it's overkill for this little guy here.

**Comment by oxaudo** [comment]
@damassi - I'm closing this for now. Not a clear improvement here and seems like we might want to review our images more holistically at some point anyways.
